packages/cursorless-vscode/src/ide/vscode/hats/VscodeHatRenderer.ts

```diff
@@ -307,10 +315,25 @@ export default class VscodeHatRenderer {
     return isOk;
   }
 
-  private constructColoredSvgDataUri(originalSvg: string, color: string) {
+  private constructColoredSvgDataUri(svgInfo: SvgInfo, color: string) {
+    const { svg: originalSvg, svgWidthPx, originalViewBoxWidth } = svgInfo;
+    let replacementEquals, replacementColon: string;
+    // If color contains a dash, the second part is a stroke.
+    // If you are code spelunking and have found this undocumented (and thus potentially transient) feature,
+    // please subscribe to https://github.com/cursorless-dev/cursorless/pull/1810
+    // so that you can be notified if/when it changes or is removed.
+    if (color.includes("-")) {
+      const [fill, stroke] = color.split("-");
+      const strokeScaleFactor = (0.7 * originalViewBoxWidth) / svgWidthPx;
```

Comment:
This should probably be called `strokeWidth`

Reply:
indeed, done